### ISSUE TITLE
3D Viewport controls are not working properly in godot 3.2.2 after fresh installion

### ISSUE THREAD

**Issue by adityabmv27** [godotengine/godot#40449]
**Godot version:**
3.2.2

**OS/device including version:**
Windows 10 

**Issue description:**
I installed Godot For the first time and when I try to follow an online instructor on how he controls the 3D viewport nothing seems to work in my system. For Example -
1) Instructor said to use Middle-Mouse-Button for orbiting but for me it does not work. But Orbiting worked when I HELD RIGHT-CLICK.
2)Instructor said to use SHIFT + Middle-Mouse-Button for panning but for me it does not work at all. I am still unable to pan in GODOT

Please Help I really want to learn and use GODOT.

**Comment by Calinou** [comment]
> But Orbiting worked when I HELD RIGHT-CLICK.

Holding the right mouse button will toggle *freelook* mode in 3D, which is different from orbiting around a fixed point. (That said, the middle and right mouse button do the same thing in 2D, which is panning.)

PS: Godot is written as a proper noun, not in uppercase. It's not an acronym.

**Comment by adityabmv27** [comment]
I have checked my mouse and it seems that the problem is of my mouse, due to COVID-19 in my area there is no new mouse available for purchase. Now I have a laptop with touchpad, I tried to enable middle mouse button in touchpad and it seems that my laptop's touchpad is not supported. Can I control the 3D Viewport with only touchpad ?
If YES then How ?

In the previous @Calinou has suggested me enable 3-Button Mouse Emulation Mode, I have enabled it but after enabling it How do I control the 3D Viewport ?

I hope I get the answer quickly so that I can start learning 3D Game Development in Godot soon.

Thank You :)
